### PR TITLE
[TECH-673]Sign-up Page UI Observations changes

### PR DESCRIPTION
**Pull request by PoornaPanda9785**

### CODE REVIEW

File: src/pages/sign-up/SignUp.scss [hunk 1]
```diff
@@ -25,3 +25,7 @@
   @apply text-3xl lg:text-4xl text-start select-none p-5 text-white;
   font-family: 'Lato', sans-serif;
 }
+input::-webkit-inner-spin-button {
```
Comment (JeanMariaJose) [inline src/pages/sign-up/SignUp.scss:28]: don't we need a .input? Does this affect all input fields? You only need to remove up and down arrow for phone number button. make sure this is not generic and wont affect the whole project (generic one's can affect the whole project . Earlier I had an issue like that)

File: src/pages/sign-up/SignUp.tsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ const schema = Yup.object().shape({
     .matches(/^[aA-zZ\s]+$/, 'Please enter valid name')
```
Comment (JeanMariaJose) [inline src/pages/sign-up/SignUp.tsx:28]: We need to rephrase first name and last name error message right?

File: src/pages/sign-up/SignUp.tsx [hunk 3]
```diff
@@ -28,6 +28,7 @@ const schema = Yup.object().shape({
...
     .required('*Lastname is required'),
   password: Yup.string().required('*Password is required'),
+  phoneNumber: Yup.number().min(100000000000, 'Phone number must be 12 digits'),
```
Comment (JeanMariaJose) [inline src/pages/sign-up/SignUp.tsx:31]: 1. Do we need to make phone number mandatory . Could you pls confirm with Simy and do change here if yes
2. This validation won't work in country code right? Don't we need a validation for country code and number of   digits? (You can also confirm that with Simy)
3. In TECH-673 , there is one more change like  add a space between text 'PhoneNumber' -> Phone Number. Can you pls do that change too?

**Review by JeanMariaJose — CHANGES_REQUESTED** [review]

File: src/pages/sign-up/SignUp.tsx (hunk 3 above)
Comment (PoornaPanda9785) [reply src/pages/sign-up/SignUp.tsx:31]: Hello yes I did the changes it's not reflecting due to some issues in my git so i'll amend and do these changes again
For the country code in the I think they have planned to do it later as the excel shared by QA there it mentioned. I'll check with Simi again. And for this using Yup it is making mandatory without using .required also.

File: src/pages/sign-up/SignUp.scss (hunk 1 above)
Comment (PoornaPanda9785) [reply src/pages/sign-up/SignUp.scss:28]: Yes it is nothing but removing the arrow button. let me check if it is affecting any other.

**Review by JeanMariaJose — APPROVED** [review]

**Pull request merged**